### PR TITLE
Google api nearby places list added to database.

### PR DESCRIPTION
**Pull request by Priyankabaride**

### CODE REVIEW

File: lib/dashboard-widgets/home_screen.dart [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
 import 'package:visitplusapp/dashboard-widgets/CategoryIcon.dart';
 import 'package:visitplusapp/dashboard-widgets/doctorCard.dart';
 
 import 'package:visitplusapp/dashboard-widgets/bottom_navigation_bar.dart';
-import 'package:visitplusapp/authentication-widgets/signin_screen.dart';
 import 'package:visitplusapp/doctor-widgets/doctor_profile_screen.dart';
+import 'package:visitplusapp/map%20widgets/map_screen.dart';
```
Comment (pramodbhure) [inline lib/dashboard-widgets/home_screen.dart]: @Priyankabaride Have you used any space in this directory name "map%20widgets"?

**Review by pramodbhure — CHANGES_REQUESTED** [review]
LGTM, just see a minor comment

File: lib/dashboard-widgets/home_screen.dart (hunk 1 above)
Comment (Priyankabaride) [reply lib/dashboard-widgets/home_screen.dart]: yes i have map screen class in that which is used to add data to database

**Review by pramodbhure — APPROVED** [review]
LGTM!

**Pull request merged**